scripts/cybershake/run_cybershake.py

```diff
@@ -226,11 +226,14 @@ def parse_config_file(
         elif pattern == NONE:
             pass
         else:
-            if pattern == ONCE:
-                pattern = ONCE_PATTERN
-            if pattern not in tasks_with_pattern_match.keys():
-                tasks_with_pattern_match.update({pattern: []})
-            tasks_with_pattern_match[pattern].append(proc)
+            if isinstance(pattern, str):
+                pattern = [pattern]
+            for i in pattern:
```

Comment:
i should have a better name e.g. subpattern or match

Reply:
I agree. I couldn't think of a good one. Subpattern is a good one.